scripts/run-tests.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
```

Comment:
Always use [`set -e`](http://stackoverflow.com/q/19622198/161278) in test files so it fails correctly, otherwise this file will always succeed.